packages/dashmate/README.md

```diff
@@ -411,17 +431,17 @@ dashmate config set --config=testnet_2 group testnet
 dashmate group default testnet
 ```
 
-To start the group of nodes, ports and other required options need to be updated.
+Ports and other required options need to be updated to avoid port collisions before starting the group of nodes.
 
 ### Development
 
-To start a local dash network, the `setup` command with the `local` preset can be used to generate configs, mine some dash, register masternodes and populate the nodes with the data required for local development.
+To start a local dash network, the `setup` command with the `local` preset can be used to generate configs, mine some tDash, register masternodes and populate the nodes with the data required for local development.
```

Comment:
@strophy It looks like we have a serial comma in the re-index section so one might be needed here.
```suggestion
To start a local dash network, the `setup` command with the `local` preset can be used to generate configs, mine some tDash, register masternodes, and populate the nodes with the data required for local development.
```